cg-spec/editors_draft.html

```diff
@@ -1420,6 +1420,112 @@ <h2>isTRIPLE</h2>
+
+        <pre data-transform="updateExample"
+             data-content-type="application/x-sparql-star-query"
+             class="nohighlight example"
+        >
+          <!--
+          SELECT ?t WHERE {
+            ?s ?p ?o .
+            BIND( TRIPLE(?s,?p,?o) AS ?t )
+            FILTER( isTRIPLE(SUBJECT(?)) )
+          }
+          -->
+        </pre>
+
+        <p>Instead of accessing the subject of the triples in the FILTER after the BIND clause, as was done in the previous query, a semantically equivalent query may apply the FILTER directly on the variable&nbsp;<code>?s</code>, as follows:</p>
+
+        <pre data-transform="updateExample"
+             data-content-type="application/x-sparql-star-query"
+             class="nohighlight example"
+        >
+          <!--
+          SELECT ?t WHERE {
+            ?s ?p ?o .
+            BIND( TRIPLE(?s,?p,?o) AS ?t )
+            FILTER( isTRIPLE(?s) )
+          }
+          -->
+        </pre>
+
+        <p>While the triples that the previous example queries bind to variable&nbsp;<code>?t</code> occur in the queried data, the <a href="#triple-function">TRIPLE function</a> can be used to construct triples that may not be in the data. For instance, the following query uses the subject of each <a>asserted triple</a> in the <a>default graph</a> of the queried <a>RDF-star dataset</a> to construct another triple, and bind it to the variable&nbsp;<code>?t</code>. Notice that the result of this query contains as many <a>SPARQL-star solution mappings</a> as there are asserted triples in the queried graph, and there may be duplicates in the result if multiple asserted triples have the same subject.</p>
+
+        <pre data-transform="updateExample"
+             data-content-type="application/x-sparql-star-query"
+             class="nohighlight example"
+        >
+          <!--
+          PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+          PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+
+          SELECT ?t WHERE {
+            ?s ?p ?o .
+            BIND( TRIPLE(?s,rdf:type,rdfs:Resource) AS ?t )
+          }
+          -->
+        </pre>
+
+        <p>There are cases where the effects of using the new TRIPLE function are not immediately obvious just from its definition, particularly when considering the interplay of this function with the <a data-cite="SPARQL11-QUERY#func-bnode">BNODE function</a> when used within <a data-cite="SPARQL11-QUERY#queryDataset">GRAPH clauses</a>. Some of these cases are discussed below.</p>
```

Comment:
For what it's worth I find these cases contrived. It's basically an explanation that says: "if you write this weird artificial construct, here's what happens". All of the elements mentioned are well-understood parts of the language, how they interact is not in any way ambiguous, it's complex only because it is doing some non-trivial combination of language features. 

I have no objection to including it in the editor draft, and the explanation as it stands is a clear and concise one, but I'd be a little wary of adding as part of an introductory / primer text.

Reply:
@jeenbroekstra I agree. Note that there was a request to have a discussion of such contrived cases in this part of the spec, and my intention is to keep it here. The only thing that I would move to the overview/primer part of the report are the three examples that come before the discussion of these contrived cases (i.e., Examples 12-14).